04Lesson/01Homework/css/style.css

```diff
@@ -1 +1,32 @@
+#links {
 
+  color:white;
+  background-color:black;
+  height:50px;
+
+
+
+}
+a {
+  text-decoration:none; color:white;
+}
+#employee {
+  float: left;
+  background-color:green;
+  width:452px;
+  padding:40px;
+  /*margin:10px;*/
+  margin-left: 100px;
+  margin-top: 10px;
+}
+#job {
+  float:right;
```

Comment:
float both of these sections the same direction so you can remove that large white gap between the two.